embeddings/data/datamodule.py

```diff
@@ -1,17 +1,17 @@
 import abc
-from typing import Any, Dict, Generic, List, Optional, Sequence, TypeVar, Union
+from typing import Any, Dict, Generic, List, Optional, Sequence, Type, TypeVar, Union
 
 import datasets
 import pytorch_lightning as pl
-from dataset.arrow_dataset import Dataset
 from datasets import ClassLabel, DatasetDict
+from datasets.arrow_dataset import Dataset
```

Comment:
```suggestion
from datasets import ClassLabel, DatasetDict, Dataset
```

I a new version of Datasets it is necessary